app/views/insured/families/_enrollment.html.erb

```diff
@@ -243,6 +262,8 @@
   // For when families home page is refreshed when user on it
   document.addEventListener("DOMContentLoaded", function() {
     var enrollmentToggleCheckbox = document.getElementById("display_all_enrollments");
-    enrollmentToggleCheckbox.addEventListener('click', toggleDisplayEnrollments);
+    if (enrollmentToggleCheckbox != null || enrollmentToggleCheckbox != undefined) {
```

Comment:
not a problem really, but using `!=`, an `undefined` `enrollmentToggleCheckbox` will never reach the second comparison. Could either omit, or use `!==` for these comparisons.